pyanaconda/installclass.py

```diff
@@ -64,7 +64,7 @@ class BaseInstallClass(object):
     default_autopart_type = DEFAULT_AUTOPART_TYPE
 
     # help
-    help_folder = "/usr/share/anaconda/help"
+    help_folder = "/usr/share/anaconda/help/rhel"
```

Comment:
The RHEL content is basically the default help content for all variants, unless it specifies its own, so that's why it's defined this in the `BaseInstallClass`.

It could be also possible to put the help content to `default` subfolder, but having it in a `rhel` subfolder seemed more self-describing to me, as it is really just the converted RHEL installation guide without any variant specific changes that serves as the default help content.